libs/langchain/langchain/chat_models/litellm.py

```diff
@@ -348,7 +353,10 @@ def _create_chat_result(self, response: Mapping[str, Any]) -> ChatResult:
             )
             generations.append(gen)
         token_usage = response.get("usage", {})
-        llm_output = {"token_usage": token_usage, "model_name": self.model_name}
+        set_model_value = self.model
+        if self.model_name is not None:
+            set_model_value = self.model_name
+        llm_output = {"token_usage": token_usage, "model": set_model_value}
```

Comment:
`llm_output = {"token_usage": token_usage, "model": self.model_name or self.model}`
can be simpler. The same for L263